main.go

```diff
@@ -85,47 +55,46 @@ func main() {
 	os.Exit(_main(env))
 }
 
-func _main(env envConfig) int {
+func _main(env utils.EnvConfig) int {
 	ctx := context.Background()
 	ctx = cloudevents.WithEncodingStructured(ctx)
 
 	p, err := cloudevents.NewHTTP(cloudevents.WithPath(env.Path), cloudevents.WithPort(env.Port))
 	if err != nil {
 		log.Fatalf("failed to create client, %v", err)
 	}
+	// Create CloudEvents client
 	c, err := cloudevents.NewClient(p)
 	if err != nil {
 		log.Fatalf("failed to create client, %v", err)
 	}
+	// Start CloudEvents receiver
 	log.Fatal(c.StartReceiver(ctx, gotEvent))
 
 	return 0
 }
 
+// gotEvent processes an incoming CloudEvent
 func gotEvent(event cloudevents.Event) error {
 	var shkeptncontext string
 	_ = event.Context.ExtensionAs("shkeptncontext", &shkeptncontext)
 
-	if event.Type() == keptnv2.GetTriggeredEventType(keptnv2.GetSLITaskName) {
-		return processEvent(event)
-	}
-
-	logger := keptncommon.NewLogger(shkeptncontext, event.Context.GetID(), "prometheus-service")
-
 	keptnHandler, err := keptnv2.NewKeptn(&event, keptncommon.KeptnOpts{})
 
 	if err != nil {
 		return fmt.Errorf("could not create Keptn handler: %v", err)
 	}
 
+	logger := keptncommon.NewLogger(shkeptncontext, event.Context.GetID(), utils.ServiceName)
+
 	if err = eventhandling.NewEventHandler(event, logger, keptnHandler).HandleEvent(); err != nil {
 		return err
 	}
 	return nil
 
```

Comment:
I know you didn't touch this but we can simplify this as 
```suggestion
	return eventhandling.NewEventHandler(event, logger, keptnHandler).HandleEvent()
```